Analysis/src/Coda3EventDecoder.cc

```diff
@@ -0,0 +1,422 @@
+#include "Coda3EventDecoder.h"
+#include "THaCodaFile.h"
+#include "QwOptions.h"
+
+#include <vector>
+#include <ctime>
+
+#include "TError.h"
+
+// Encoding Functions
+
+std::vector<UInt_t> Coda3EventDecoder::EncodePHYSEventHeader()
```

Comment:
Starting to think about how to make this more "robust" in terms of QwMockDataGeneration.